deploy/libproject/deploy.rb

```diff
@@ -115,6 +115,7 @@ def setup_and_enter_docker(cmd_name, opts)
       --account #{opts.account}
       --project #{opts.project}
       #{opts.promote ? "--promote" : "--no-promote"}
+      --allow_empty_tiers
```

Comment:
Please add a comment here with https://precisionmedicineinitiative.atlassian.net/browse/RW-7931 - and please also add a reference to this change in the ticket